README.md

```diff
@@ -71,30 +73,33 @@ from the received data after a time period.
 2. DESTimes - This folder contains the DESTimes library files.
 3. vendor   - This folder contains third party vendor files.
     + include
-        * iestream.hpp
-        * NDTime.hpp 
+        * iestream.hpp 
 
 		 
 **src** - This folder contains the source files(.cpp).
 
 1. message 
     + message.cpp
 2. top_model
-    + main.cpp	 		
+    + main.cpp	 	
+3. text_filter.cpp [to modify the output]	
 
 **test** - This folder contains files for the unit tests.
 
 1. data  - This folder contains the input and output files for the unit tests.
     + receiver  - This folder contains input and output data for receiver
         * receiver_input\_test.txt
 		* receiver_test\_output.txt
+		* output.txt
 	+ sender - This folder contains input and output data for sender.
 	    * sender_input_test_ack\_In.txt
 	    * sender_input_test_control\_In.txt
 	    * sender_test\_output.txt
+	    * output.txt
 	+ subnet  - This folder contains input and output data for subnet.
 	    * subnet_input\_test.txt
 	    * subnet_test\_output.txt
+	    * output.txt
```

Comment:
add information about the new formatted output file path, which u added as part of task d

Reply:
resolved